platform/core.osgi/src/org/netbeans/core/osgi/OSGiRepository.java

```diff
@@ -119,8 +118,17 @@ public void removeLayersFor(List<Bundle> bundles) {
     private static final class SFS extends MultiFileSystem implements LookupListener {
 
         static {
-            @SuppressWarnings("deprecation") Object _1 = FileSystem$Environment.class; // FELIX-2128
```

Comment:
[FELIX-2128](https://jira.apache.org/jira/browse/FELIX-2128) nine years ago! AFAICT the intent here was to load some classes which we expected to be loaded anyway. If `FileSystem$Environment` and `FileSystemCapability` have been removed from the active codebase then I presume these two class loads could simply be deleted.

[FELIX-3477](https://jira.apache.org/jira/browse/FELIX-3477) BTW for the `LookupEvent` load, apparently. I do not really remember anything about these issues.